app/assets/javascripts/govuk_publishing_components/polyfills/nodelist-to-array.js

```diff
@@ -0,0 +1,19 @@
+// helper to loop over nodes and create an array for cross-browser compatibillity
+// if browser support Array.from we'll call that
+// else iterate over the array with a good'ol for-loop
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Polyfills = window.GOVUK.Polyfills || {};
+
+(function (Polyfills) {
```

Comment:
What's your thinking of doing this instead of using the polyfill in GOV.UK Frontend? Not saying it's the wrong approach since I'm not 100% confident with our polyfills but interested to hear your thoughts.

Reply:
it was my first point of call, but the polyfill in the frontend is attached to the global scope so we can't initialise in in publishing components. additionally it doesn't work on IE8